### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by MartoMcfly**
## Summary
1. List All Convert Pairs
2. Send quote request (USER_DATA)
3. Accept Quote (TRADE)
4. Order status (USER_DATA)
5. Get Convert Trade History (USER_DATA)

## Additional Information
1. List All Convert Pairs
2. Send quote request (USER_DATA)
3. Accept Quote (TRADE)
4. Order status (USER_DATA)
5. Get Convert Trade History (USER_DATA)

### CODE REVIEW

**Comment by MartoMcfly** [comment]
Hello @tiagosiebler there seems to be a problem with my merge request, could you give me a hand?

**Comment by MartoMcfly** [comment]
@tiagosiebler Hello, could you give me a hand on why the circle test is not valid?

File: src/main-client.ts [hunk 1]
```diff
@@ -310,6 +315,36 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/account/apiRestrictions');
   }
 
+  acceptQuoteRequest(
+    params: EnableConvertSubAccountParams,
+  ): Promise<any>; {
+    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
+  }
```
Comment (tiagosiebler) [inline src/main-client.ts:322]: Typo here is breaking the code, should be:
```suggestion
  acceptQuoteRequest(
    params: EnableConvertSubAccountParams,
  ): Promise<any> {
    return this.postPrivate('sapi/v1/convert/acceptQuote', params);
  }
```

File: src/main-client.ts [hunk 2]
```diff
@@ -310,6 +315,36 @@ export class MainClient extends BaseRestClient {
...
+    params: GetOrderStatusParams,
+  ): Promise<any>; {
+    return this.getPrivate('sapi/v1/convert/orderStatus', params);
+  }
+
+  getConvertTradeHistory(
+    params: GetConvertTradeHistoryParams,
+  ): Promise<any>; {
+    return this.getPrivate('sapi/v1/convert/tradeFlow', params);
+  }
```
Comment (tiagosiebler) [inline src/main-client.ts:346]: Same for these

**Comment by MartoMcfly** [comment]
Thank you very much for the corrections, I have made the requested changes
________________________________
De: Tiago ***@***.***>
Enviado: jueves, 27 de julio de 2023 11:15 a. m.
Para: tiagosiebler/binance ***@***.***>
Cc: Martin Pelaez ***@***.***>; Author ***@***.***>
Asunto: Re: [tiagosiebler/binance] Patch 1 (PR #350)


@tiagosiebler commented on this pull request.

________________________________

In src/main-client.ts<https://github.com/tiagosiebler/binance/pull/350#discussion_r1275988864>:

> +  enableConvertSubAccount(
+    params: AcceptQuoteRequestParams
+  ): Promise<any>; {
+    return this.postPrivate('sapi/v1/broker/subAccount/convert', params);
+  }
+
+  convertQuoteRequest(
+    params: ConvertQuoteRequestParams,
+  ): Promise<any>; {
+    return this.postPrivate('sapi/v1/convert/getQuote', params);
+  }
+
+  getOrderStatus(
+    params: GetOrderStatusParams,
+  ): Promise<any>; {
+    return this.getPrivate('sapi/v1/convert/orderStatus', params);
+  }
+
+  getConvertTradeHistory(
+    params: GetConvertTradeHistoryParams,
+  ): Promise<any>; {
+    return this.getPrivate('sapi/v1/convert/tradeFlow', params);
+  }


Same for these

**Pull request closed without merging**